### ISSUE TITLE
Cannot push pages that have comments

### ISSUE THREAD

**Issue by jmcdo29** [skuid/skuid-sfdx#11]
If a page is pulled from a Salesforce organization and it has single line comments in JavaScript (using `//`), then when the page gets serialized to XML, all of the JavaScript becomes a single line for that snippet. If there is a single line comment in that snippet, any code after that comment becomes unusable. Then when the page is pushed back up to Salesforce, the snippet stays as a single line in the XML file, meaning that the page becomes unusable due to the missing JavaScript functionality.

I would expect that when the page is de-serialized on the server that the page can still run as expected. Not sure where the fix is yet, but I'd be willing to help if possible.

Sample of jsitem with comments:
```xml
<jsitem location="inlinesnippet" name="Convert Lead" cachelocation="false">var dfd = $.Deferred();  // Set timeout to give the UI enough time to block the UI window.setTimeout(function(){     LeadConvert.convertLead(dfd); }, 500);  return dfd.promise();</jsitem>
```

**Comment by Rnhatch** [comment]
Thanks for the report.  We will look into this.

**Comment by pfkong** [comment]
@Rnhatch I've been experiencing this too and it's a blocker in my page versioning/backup process. Will be following this thread :)

**Comment by zachelrath** [comment]
Addressed in release [0.3.0](https://github.com/skuid/skuid-sfdx/releases/tag/0.3.0)
